src/System.Memory/src/ApiCompatBaseline.uapaot.txt

```diff
@@ -1,4 +1,37 @@
 Compat issues with assembly System.Memory:
 CannotRemoveBaseTypeOrInterface : Type 'System.Memory<T>' does not implement interface 'System.IEquatable<System.Memory<T>>' in the implementation but it does in the contract.
 CannotRemoveBaseTypeOrInterface : Type 'System.ReadOnlyMemory<T>' does not implement interface 'System.IEquatable<System.ReadOnlyMemory<T>>' in the implementation but it does in the contract.
-Total issues: 2
+MembersMustExist : Member 'System.MemoryExtensions.Trim(System.Memory<System.Char>)' does not exist in the implementation but it does exist in the contract.
```

Comment:
FYI, @MichalStrehovsky, @jkotas - to remove whenever feasible (when we get the relevant dependencies update).